packages/@aws-cdk/cloudformation-include/lib/cfn-include.ts

```diff
@@ -44,24 +45,13 @@ export class CfnInclude extends core.CfnElement {
     for (const logicalId of Object.keys(this.template.Resources || {})) {
       this.getOrCreateResource(logicalId);
     }
+
+    // instantiate all parameters
+    for (const logicalId of Object.keys(this.template.Parameters || {})) {
+      this.createParameter(logicalId);
+    }
```

Comment:
This needs to be moved above the `getOrCreateResource()` loop, because parameters can be referenced by resources (it's not important right now, but it will be in the future).